packages/aws-rfdk/lib/deadline/test/render-queue.test.ts

```diff
@@ -1134,7 +1243,7 @@ describe('RenderQueue', () => {
                   },
                 ],
               },
-              `" --render-queue "http://renderqueue.${ZONE_NAME}:8080" \n` +
+              '" --render-queue "http://renderqueue.renderfarm.local:8080" \n' +
```

Comment:
Should we have changed this? The previous code avoided duplication and more clearly articulated the expected semantics of the behavior.

Same feedback applies for line 1417 below